timemachine/lib/potentials.py

```diff
@@ -63,81 +63,6 @@ def unbound_impl(self, precision):
         return custom_ops.FanoutSummedPotential(impls)
 
 
-# should not be used for Nonbonded Potentials
-# class InterpolatedPotential(CustomOpWrapper):
-
-#     def bind(self, params):
-#         assert self.get_u_fn().params is None
-#         self.params = params
-#         return self
-
-#     def get_u_fn(self):
-#         return self.args[0]
-
-#     def unbound_impl(self, precision):
-#         return custom_ops.InterpolatedPotential(
-#             self.get_u_fn().unbound_impl(precision),
-#             self.args[1],
-#             self.args[2]
-#         )
-
-#     def bound_impl(self, precision):
-#         return custom_ops.BoundPotential(
-#             self.unbound_impl(precision),
-#             self.params
-#         )
-
-# class LambdaPotential(CustomOpWrapper):
-
-#     def bind(self, params):
-#         assert self.get_u_fn().params is None
-#         self.params = params
-#         return self
-
-#     # pass through so we can use the underlying methods
-#     def __getattr__(self, attr):
-#         return getattr(self.args[0], attr)
-
-#     def get_u_fn(self):
-#         return self.args[0]
-
-#     def set_N(self, N):
-#         self.args[1] = N
-
-#     def get_N(self):
-#         return self.args[1]
-
-#     def get_multiplier(self):
-#         return self.args[3]
-
-#     def get_offset(self):
-#         return self.args[4]
-
-#     def unbound_impl(self, precision):
-#         return custom_ops.LambdaPotential(
-#             self.args[0].unbound_impl(precision),
-#             *self.args[1:]
-#         )
-
-#     def bound_impl(self, precision):
-#         u_params = self.get_u_fn().params
-#         return custom_ops.BoundPotential(
-#             self.unbound_impl(precision),
-#             self.params
-#         )
-
-# class Shape(CustomOpWrapper):
-
-#     def get_N(self):
-#         return self.args[0]
-
-#     def get_a_idxs(self):
-#         return self.args[1]
-
-#     def get_b_idxs(self):
-#         return self.args[2]
-
-
```

Comment:
Should `InterpolatedPotential` be moved to `attic/` (along with the custom ops implementation) or just deleted?

Reply:
Just delete it. I can't see a future use for this at all.

Reply:
Fully deleted in 68bff6e